### PR TITLE
Fixes #26255 - fix boolean search definition

### PR DESCRIPTION
**Pull request by ares**

### CODE REVIEW

**Comment by jlsherrill** [comment]
@ares is this the right issue #

**Comment by ares** [comment]
Oops sorry, fixed the number in the commit message, name of PR and the PR link in redmine issue.

File: db/seeds.d/102-organizations.rb [hunk 1]
```diff
@@ -12,12 +12,12 @@
...
-end
+#if ENV['SEED_ORGANIZATION']
+#  if Setting['db_pending_seed']
+#    admin = User.where(:login => ENV['SEED_ADMIN_USER'].present? ? ENV['SEED_ADMIN_USER'] : 'admin').first
+#    if admin && admin.default_organization.nil?
+#      admin.default_organization = Organization.find_by(:name => ENV['SEED_ORGANIZATION'])
+#      admin.save!
+#    end
+#  end
+#end
```
Comment (jlsherrill) [inline db/seeds.d/102-organizations.rb]: Whoops not again! :)

Comment (ares) [reply db/seeds.d/102-organizations.rb]: Oh damn, I need to figure if I still need this workaround :-) sorry, will push the right fix later, probably tomorrow. Quick fixes turns to be time consuming if one does not pay attention.

**Comment by ares** [comment]
updated

**Comment by jlsherrill** [comment]
[test katello]

**Pull request merged**